app/src/main/java/agency/five/codebase/android/movieapp/ui/favorites/FavoritesViewState.kt

```diff
@@ -8,5 +8,5 @@ data class FavoritesMovieViewState(
 )
 
 data class FavoritesViewState(
-    val favoriteMovies: List<FavoritesMovieViewState>
+    val favoriteMovies: List<FavoritesMovieViewState> = listOf()
```

Comment:
Remove default value.
If you need it as a default value for a view state you have 2 choices:
1. Initialise it on the place where you need it  – `MutableStateFlow(FavoriteViewState(emptyList()))`
2. Use an EMPTY constant in the data class
```kotlin
data class FavoritesViewState(...) {
    companion object {
        const val EMPTY: FavoriteViewState = FavoriteViewState(emptyList())
    }
}
```
usage:
```kotlin
MutableStateFlow(FavoritesViewState.EMPTY)
```